### ISSUE TITLE
Test on a new string of gpt2 fine tuned

### ISSUE THREAD

**Issue by vyaslkv** [huggingface/transformers#5947]
I fine tuned GPT2 on my own dataset now I have fine tuned model how can I test it on new string??

GPT-2/GPT and causal language modeling
The following example fine-tunes GPT-2 on WikiText-2. We're using the raw WikiText-2 (no tokens were replaced before the tokenization). The loss here is that of causal language modeling.

```
export TRAIN_FILE=/path/to/dataset/wiki.train.raw
export TEST_FILE=/path/to/dataset/wiki.test.raw

python run_language_modeling.py \
    --output_dir=output \
    --model_type=gpt2 \
    --model_name_or_path=gpt2 \
    --do_train \
    --train_data_file=$TRAIN_FILE \
    --do_eval \
    --eval_data_file=$TEST_FILE
```

**Comment by vyaslkv** [comment]
I have these files generated
<img width="1389" alt="Screenshot 2020-07-22 at 12 39 19 AM" src="https://user-images.githubusercontent.com/33617789/88096141-d1f06a80-cbb3-11ea-8468-9284fb074bef.png">

**Comment by vyaslkv** [comment]
can you please provide a code snippet

**Comment by patil-suraj** [comment]
Not sure what you mean by test, do you want to generate using the model or calculate loss and perplexity for the test data ?

For generation you can use the `.generate` method. This [blog post ](https://huggingface.co/blog/how-to-generate) explains generate very well.

**Comment by vyaslkv** [comment]
I want to load this model and tokenizer for testing I went through this post it is cool

**Comment by vyaslkv** [comment]
got it thanks :)

**Comment by LysandreJik** [comment]
Closing this as the issue is solved. Feel free to re-open if you still face issues.
